bundle/bundle.go

```diff
@@ -942,6 +975,14 @@ func (b *Bundle) readData(key []string) *interface{} {
 	return &child
 }
 
+// GetBundleType returns the type of the bundle.
+func (b *Bundle) GetBundleType() string {
```

Comment:
nit: This function name is a bit redundant, I'd shorten it to just `Type()`.

Reply:
Done